src/components/app/ListOfPublishedProfiles.js

```diff
@@ -254,12 +259,17 @@ export class ListOfPublishedProfiles extends PureComponent<Props, State> {
         <>See and manage all your recordings ({profilesRestCount} more)</>
```

Comment:
It looks like there is another string to localize here.

Reply:
Oh yes, I missed it, didn't find this in the UI so couldn't test also. @canova  Review this portion, please. I can't mention Flodolo.